tests/codeception/functional/Install/Content/FlexiblePageCest.php

```diff
@@ -151,4 +151,66 @@ public function testMobileMenu(FunctionalTester $I) {
     $I->click('.hb-main-nav__link');
   }
 
+    /**
+   * I can create a page with a spotlight slider.
+   */
+  public function testSpotlightSlider(FunctionalTester $I) {
+    $I->logInWithRole('contributor');
+    $I->amOnPage('node/add/hs_basic_page');
+    $I->fillField('Title', 'Demo Basic Page');
```

Comment:
I suggest using the faker service. My team has moved over to making these strings as dynamic as possible for a couple reasons.
1. more random testing
2. avoids cache issues

the second being the most beneficial. Because if you have two tests that have the same node title, sometimes the cache gets messy or you get incorrect responses in the tests.

Reply:
I added in the faker service and used it for the title here.